src/overlays/actors/ovl_Dm_Bal/z_dm_bal.c

```diff
@@ -98,34 +98,34 @@ void func_80C1EAE8(DmBal* this, PlayState* play) {
             }
         } else if (D_80C1F2C0 == 3) {
             if (Animation_OnFrame(&this->skelAnime, 0.0f)) {
-                this->unk_336 = 1;
+                this->keepEyesShut = true;
             } else if (Animation_OnFrame(&this->skelAnime, 29.0f)) {
-                this->unk_336 = 0;
+                this->keepEyesShut = false;
                 this->eyeIndex = 0;
             }
         }
         Cutscene_ActorTranslateAndYaw(&this->actor, play, actionIndex);
         this->actor.home.pos = this->actor.world.pos;
     } else {
-        this->unk_336 = 0;
+        this->keepEyesShut = false;
         this->eyeIndex = 0;
         D_80C1F2C0 = 0x63;
     }
 }
 
 void func_80C1EC60(DmBal* this, PlayState* play) {
-    f32 temp_fv1_2;
+    f32 temp;
```

Comment:
```suggestion
    f32 scaleX;
```
Seems reasonable name, even if it does get squared to be the real scaleX